kfp/transform_workflows/universal/doc_id/Makefile

```diff
@@ -1,6 +1,8 @@
-# Define the root of the local git clone for the common rules to be able 
-# know where they are running from.
-REPOROOT=../../../..
+REPOROOT ?= ${CURDIR}/../../../../
+# venv is common to all workflows
+VENV_ACTIVATE ?= ${REPOROOT}/kfp/transform_workflows/venv/bin/activate
+PYTHON ?= python3
```

Comment:
I think it should be python, not python3 as in other places

Reply:
Fixed. Thanks

Reply:
This still shows python3